docs/TrafficSplitting.md

```diff
@@ -0,0 +1,163 @@
+# Traffic Splitting
+
+The purpose of this section of the kn documentation is to describe the traffic splitting operations with examples.
+
+The traffic splitting block for a Service may contain multiple targets. A target contains fields which can be operated by `kn` to define the
+behavior when a Service receives the traffic. One can define what percentage of the traffic should be routed to a particular Revision and also
+optionally create a custom URL for referenced Revision.
+
+`kn` client can work with following properties of a traffic target defined in traffic block of service.
+
+- **RevisionName**: Name of a specific the Revision to which to send a portion of traffic
```

Comment:
Looks like there's an extra `the` in `Name of a specific the Revision`?

Reply:
Took off that section from docs altogether.